src/models/sqlOutputContentProvider.ts

```diff
@@ -597,14 +597,21 @@ export class SqlOutputContentProvider {
                                     ),
                                 isError: false, // Elapsed time messages are never displayed as errors
                             });
-                        const tabState =
-                            Object.keys(
-                                this._queryResultWebviewController.getQueryResultState(
-                                    uri,
-                                ).resultSetSummaries,
-                            ).length > 0
```

Comment:
If there's an error, wouldn't the length of `resultSetSummaries` be `0` leading to `tabState` set to `Messages`?

Reply:
It seems like that is not the case, for example if I run this query:
```
SELECT 1/0
```

currently it gets set to the Results tab showing this (even though there is an error message in the messages pane):

![Screenshot 2025-03-13 130251](https://github.com/user-attachments/assets/89c10ba7-a29a-4a1a-b4a0-d5a5818b4c3c)
![Screenshot 2025-03-13 130505](https://github.com/user-attachments/assets/fd65a778-1786-4dec-8dba-3fa64090ddff)


I also think it makes sense to explicitly check for errors rather than rely on the resultSetSummary length

Reply:
ah I see, in this case resultSummaries would have a length of 1, makes sense.